setup.py

```diff
@@ -4,7 +4,6 @@
 from setuptools import find_packages, setup
 
 if __name__ == "__main__":
-
```

Comment:
nit: This diff looks extraneous?

Reply:
I assume this is an inadvertent `black` reformat.

Reply:
Yep, just a black thing